queries.sql

```diff
@@ -1,3 +1,23 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+SELECT * from animals WHERE name LIKE '%mon'; 
+
+SELECT name from animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth < '2019-01-01';
+
+SELECT name from animals WHERE neutered = true and escape_attempts < 3;
+
+SELECT date_of_birth from animals WHERE name = 'Agumon' or  name = 'Pikachu';
+
+SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
+
+SELECT * from animals WHERE neutered = true;
+
+SELECT * from animals WHERE name = 'Gabumon';
```

Comment:
- Kindly, find all animals that are **not** named Gabumon, so your where condition has to be `name <> 'Gabumon'  to fit the requirement 👍  
![Screenshot 2023-07-11 100419](https://github.com/jojo987N/vet-clinic/assets/52010789/0acd61b9-860b-451a-9a95-486526a63408)